checklist.md

```diff
@@ -12,18 +12,18 @@ All services should be independent.
 We shouldn't have Strategy and call its methods in CsvFileReaderService, or we shouldn't have CsvFileWriterService and call its methods in the Strategy class.
 
 Let's create `Main` class with `main()` method to show how the program works.
-Make all services independent and call them in the right order in `main()` method step by step (the result of previous service method should be the input for next one)
+Make all services independent and call them in the right order in `main()` method step by step (the result of previous core.basesyntax.db.service method should be the input for next one)
```

Comment:
do not change checklist.md file

Reply:
Return it back

Reply:
I didn't modified these files myself, I don't know why they changed